frontend/src/components/mine/Compliance/MineComplianceInfo.js

```diff
@@ -0,0 +1,138 @@
+import moment from "moment";
```

Comment:
Moment is cool. Please broadcast you're addition and use of moment to the group, so that others are aware of it and also don't add other libraries to deal with time.